pkg/acsengine/defaults.go

```diff
@@ -394,6 +394,12 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 			o.KubernetesConfig.NetworkPlugin = NetworkPolicyCilium
 		}
 
+		if o.KubernetesConfig.NetworkPolicy == NetworkPolicyAzure {
+			if !common.IsKubernetesVersionGe(k8sVersion, "1.8.0") {
```

Comment:
Let's move this < 1.8.0 logic up into the above switch/case statement, as it is executing against an identical condition (NetworkPolicy  == "azure")